### ISSUE TITLE
Can't change the exchange rates - bug?

### ISSUE THREAD

**Issue by mittt** [mvarnagiris/financius#369]
I can't change the exchange rates, in the current version. This feature was present in the past. Maybe it got dropped with the material design updates.

I can provide more info if you need it. I tries it on the device that I have my data imported(on a Moto G 4G) and also on the clean installation (on Nexus 7 2012).

**Comment by azaelcarrera** [comment]
I'm having the same issue. I used to manually set an exchange rate but since the last update now I can't. It just shows 1.0
It can't even update automatically.
Hope you can fix it because I have to use the calculator every time I want to know how much money I have left. It's not much practical. :P

**Comment by mvarnagiris** [comment]
Does this happen when entering transaction?

**Comment by azaelcarrera** [comment]
No. I can change it when entering a transaction. I'm not sure if is the same for mittt. But I can't change it from the settings. Settings > Currencies > USD
It just shows 1.0 and nothing happens when I tap the refresh button. And can't change it manually.
Here's a screenshot: https://www.dropbox.com/s/gfp6okefa68xjlp/Screenshot_2015-03-11-08-48-57.png?dl=0

And this is how that affects the total on the main screen: https://www.dropbox.com/s/ud1gtryn50ccocf/Screenshot_2015-03-11-09-02-56.png?dl=0

Tarjeta de debito is in MX Pesos and 99designs and Paypal is in USD dollars. I need the total in Pesos but if you can see it counts every dollar as 1 Pesos because that's how is set on the exchange rates, which I can't change it. This didn't happen on other versions, this is since the last update.

**Comment by mittt** [comment]
I confirm, azaelcarreral is describing the same issue.

**Comment by piatrouski** [comment]
Same issue too 😐

**Comment by azaelcarrera** [comment]
Is there a way to download older versions of the app? Where?

**Comment by a3at** [comment]
So it's been 3 months since the bug was added. Is the developer still supporting the product?

**Comment by mariebogarova** [comment]
Hi there!

Were you guys able to solve this exchange currency issue?

For some currencies the app doesn't work properly and doesn't do the money conversion (for example: GBP, NZD,..). so if I wanna set my new "account" in those currencies, the money exchange rate is always 1, even it's not true (for example 1 NZD would show as 1 CHF).

I don't even see the option to set it manually, which would be totally fine.

I hope it's gonna be solved soon, I really like this simple awesome app.

Thanks for advises!
